ckanext/geocat/utils/csw_mapping.py

```diff
@@ -128,11 +146,19 @@ def get_metadata(self, csw_record_as_string, geocat_id):
                                    organization_slug=self.organization_slug,
                                    valid_identifiers=self.valid_identifiers)
         dataset_dict['owner_org'] = self.organization_slug
+
         rights = \
             _map_dataset_rights(node=root_node,
                                 terms_of_use=self.terms_of_use_graph,
                                 default_rights=self.default_rights)
 
+        # set license to rights
+        # if rights name is found in the license vocabulary
+        license = rights
+        rights_uri = get_license_uri_by_name(rights)
```

Comment:
See my comment above - there ought to always be a `rights_uri` here because (I think) the terms of use in `self.terms_of_use_graph` are a subset of the license vocabulary we are using. It should be simplest to, in `_map_dataset_rights`, check that the rights statement belongs to the license vocabulary. Then we can just set `license = rights` whatever the actual value is.

Reply:
Totally unrelated to your PR, but I just realised it while checking the code in this repo - we seem to have two copies of the terms-of-use vocabulary, `geocat-terms-of-use.xml` (used) and `geocat-terms-of-use.ttl` (unused). Could you get rid of the ttl one, and also the xml one if we end up not using it any more? Thank you!

Reply:
oh yes, you are right, the check already exists.